scripts/split-paired-reads.py

```diff
@@ -92,11 +92,15 @@ def main():
 
     infile = args.infile
 
-    check_input_files(infile, args.force)
     filenames = [infile]
     check_space(filenames, args.force)
 
     # decide where to put output files - specific directory? or just default?
+    if infile == '/dev/stdin':
+        if not (args.output_first and args.output_second):
+            print >>sys.stderr, ("Accepting input from stdin;"
```

Comment:
please put a space after the ';' - this is printed out as

    Accepting input from stdin;output filenames must be provided.

currently.